avocado/utils/multipath.py

```diff
@@ -115,6 +115,48 @@ def get_paths(wwid):
     return paths
 
 
+def get_multipath_details():
+    """
+    Get multipath details as a dictionary, as given by the command:
+    multipathd show maps json
+
+    :return: Dictionary of multipath output in json format.
+    """
+    mpath_op = process.system_output("multipathd show maps json", shell=True)
```

Comment:
Well the test should fail with "unsupported commad" output so I don't think it's necessary to add yet another parser just to support old systems noone is testing (or have hardcoded tests). Anyway yes, the shell is most certainly useless here...